yellowbrick/features/manifold.py

```diff
@@ -218,6 +220,7 @@ def __init__(
         self.alpha = alpha
         self.random_state = random_state
         self.manifold = manifold # must be set last
+        self.n_components = None
```

Comment:
`n_components` must be `2` (eventually we could make it `3`) ... this is how we set the dimensionality to draw the figure on a 2D graph. Therefore we cannot set this property, making it so the user could change the dimensionality -- we would no longer be able to draw the figure!